@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -285,99 +286,109 @@
   display: inherit;
   flex-direction: inherit;
   gap: inherit;
   background-color: inherit;
-  align-items: flex-start;
+  align-items: stretch;
+  padding: 0;
+  padding-block: var(--ax-spacing-1);
 }
 
-.navds-combobox__list-item--focus,
-.navds-combobox__list--with-hover
-  .navds-combobox__list-item:where(:not([data-no-focus="true"], .navds-combobox__list-item--new-option)):hover {
-  background-color: var(--ac-combobox-list-item-hover-bg, var(--a-surface-hover));
+.navds-combobox__list-item--focus {
   cursor: pointer;
-  border-left: 4px solid var(--ac-combobox-list-item-hover-border-left, var(--a-border-strong));
-  padding-inline-start: calc(var(--__ac-combobox-list-item-padding-inline) - 4px);
+  outline: 3px solid var(--ax-border-focus);
+  outline-offset: 2px;
 }
 
-.navds-combobox__list-item[data-no-focus="true"] {
-  cursor: not-allowed;
-  opacity: 0.4;
-}
+.navds-combobox__list-item {
+  user-select: none;
 
-.navds-combobox__list-item--selected {
-  background-color: var(--ac-combobox-list-item-selected-bg, var(--a-surface-selected));
-}
+  &[data-no-focus="true"] {
+    cursor: not-allowed;
+    opacity: 0.4;
+  }
+
+  & mark {
+    background-color: transparent;
+    font-weight: bold;
+    color: var(--ax-text-accent-strong);
+  }
 
-.navds-combobox__list-item--selected p {
-  font-weight: var(--a-font-weight-bold);
+  & svg {
+    color: var(--ax-text-accent);
+  }
 }
 
-.navds-combobox__list-item--selected.navds-combobox__list-item--focus,
-.navds-combobox__list--with-hover .navds-combobox__list-item--selected:hover {
-  background-color: var(--ac-combobox-list-item-selected-hover-bg, var(--a-surface-action-subtle-hover));
-  border-left: 4px solid var(--ac-combobox-list-item-selected-hover-border-left, var(--a-border-focus));
+.navds-combobox__list-item--selected {
+  background-color: var(--ax-bg-accent-moderate-pressedA);
+
+  & p {
+    font-weight: var(--ax-font-weight-bold);
+  }
 }
 
 .navds-combobox__list-item--new-option {
-  border-bottom: 1px solid var(--a-border-divider);
-  background: var(--a-surface-neutral-subtle);
+  border-bottom: 1px solid var(--ax-border-accent-subtleA);
+  border-radius: 0;
+  background: var(--ax-bg-accent-moderateA);
   cursor: pointer;
   justify-content: flex-start;
   gap: 0.25rem;
-}
+  margin: 0;
+  margin-block: calc(var(--ax-spacing-1) * -1);
+  padding-block: var(--ax-spacing-4);
+
+  & svg {
+    stroke: var(--ax-text-default);
+  }
 
-.navds-combobox__list--with-hover .navds-combobox__list-item--new-option:hover {
-  border-bottom: 1px solid var(--a-border-divider);
-  background: var(--a-surface-neutral-subtle-hover);
+  &:only-child {
+    border: none;
+  }
 }
 
 .navds-combobox__list-item--new-option--focus {
-  box-shadow:
-    var(--a-shadow-focus) inset,
-    var(--a-border-action) 0 0 0 5px inset;
-  border-radius: calc(var(--a-border-radius-medium) - 1px);
-}
+  border-radius: calc(var(--ax-border-radius-large) - 1px) calc(var(--ax-border-radius-large) - 1px) 0 0;
```

Comment:
Might just happen on my screen, but there is still a small gap here, even after -1px update. Think this is since the "outline" is offset -3px, making it "inset" and thus throwing the calculations a little off. 2px seems to work 👍 
![Screenshot 2024-11-18 at 13 31 34](https://github.com/user-attachments/assets/e0c43d86-82a0-4663-8b73-6240ae83649b)
![Screenshot 2024-11-18 at 13 31 42](https://github.com/user-attachments/assets/f686f8e9-0c17-4b6b-ae29-6b97518a71b4)

Reply:
I can't get it any better than this on my screen (now with -2px offset, and keeping the -1px calculation)
![image](https://github.com/user-attachments/assets/771b986f-21a4-4bb1-aa4b-7b41f26ea2db)

Reply:
This is fine 👍 You can try `calc(var(--ax-border-radius-large) - 2px) calc(var(--ax-border-radius-large) - 2px) 0 0;`, but it's probably the same.